src/DetailsView/tab-stops-requirement-instances-collapsible-content.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { AssessmentInstanceDetailsColumn } from 'DetailsView/components/assessment-instance-details-column';
+import { TabStopsRequirementResultInstance } from 'DetailsView/tab-stops-requirement-result';
+import {
+    CheckboxVisibility,
+    ColumnActionsMode,
+    ConstrainMode,
+    DetailsList,
+    IColumn,
+    Icon,
+    Link,
+} from 'office-ui-fabric-react';
+import * as React from 'react';
+import * as styles from './tab-stops-requirement-instances-collapsible-content.scss';
+
+export type TabStopsRequirementInstancesCollapsibleContentProps = {
+    instances: TabStopsRequirementResultInstance[];
+};
+export const TabStopsRequirementInstancesCollapsibleContent =
+    NamedFC<TabStopsRequirementInstancesCollapsibleContentProps>(
+        'TabStopsRequirementInstancesCollapsibleContent',
+        props => {
+            const onRenderCapturedInstanceDetailsColumn = (
+                instance: TabStopsRequirementResultInstance,
+            ): JSX.Element => {
+                return (
+                    <AssessmentInstanceDetailsColumn
+                        background={'#767676'}
+                        textContent={instance.description}
+                        headerText={'Comment:'}
+                        tooltipId={instance.id}
+                    />
+                );
+            };
+
+            const onRenderCapturedInstanceIconsColumn = (
+                instance: TabStopsRequirementResultInstance,
+            ): JSX.Element => {
+                return (
+                    <>
+                        <Link className={styles.editButton} onClick={() => {}}>
```

Comment:
For the onClick methods, might make sense to add optional parameters for now (allows you to test calling them).